module.xml

```diff
@@ -2,7 +2,7 @@
 <Export generator="Cache" version="25">
 <Document name="ZPM.ZPM"><Module>
   <Name>ZPM</Name>
-  <Version>0.9.2-SNAPSHOT</Version>
+  <Version>0.9.3-SNAPSHOT</Version>
```

Comment:
Should be 0.10.0-SNAPSHOT in v0.10.x?? Not sure why the diff is showing as this.